src/core/public/core_system.ts

```diff
@@ -163,7 +163,7 @@ export class CoreSystem {
         i18n: this.i18n.getContext(),
       });
       await this.integrations.setup();
-      const docLinks = this.docLinks.setup({ injectedMetadata });
+      this.docLinks.setup();
```

Comment:
NIT: As `setup` is now a noop, do we even have to call this?

Reply:
It's easy to forget to add it if we refactor setup later.